staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -852,13 +873,14 @@ func (v schemaCoercingConverter) ConvertFieldLabel(gvk schema.GroupVersionKind,
 	return v.delegate.ConvertFieldLabel(gvk, label, value)
 }
 
-// unstructuredSchemaCoercer does the validation for Unstructured that json.Unmarshal
+// unstructuredSchemaCoercer does the pruning for Unstructured that json.Unmarshal
```

Comment:
"unstructuredSchemaCoercer does for Unstructured what json.Unmarshal does for native types.." ?

Reply:
have reformulated it